templates/lxc-download.in

```diff
@@ -265,6 +271,7 @@ while :; do
     --rootfs)      LXC_ROOTFS="$2"; shift 2;;
     --mapped-uid)  LXC_MAPPED_UID="$2"; shift 2;;
     --mapped-gid)  LXC_MAPPED_GID="$2"; shift 2;;
+    --standard-resolver) STANDARD_RESOLVER="$2"; shift 2;;
```

Comment:
This should be:
```
--standard-resolver) STANDARD_RESOLVER="true"; shift 1;;
```

And I'd also move it up with the other flags, just under `--force-cache`